src/globus_sdk/experimental/auth_requirements_error/_serializable.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import inspect
+import typing as t
+
+T = t.TypeVar("T", bound="Serializable")
+
+
+class Serializable:
+    _EXLUDE_VARS: t.ClassVar[tuple[str, ...]] = ("self", "extra_fields", "extra")
+    extra: dict[str, t.Any]
+
+    @classmethod
+    def _supported_fields(cls) -> list[str]:
+        signature = inspect.signature(cls.__init__)
+        return [
+            name for name in signature.parameters.keys() if name not in cls._EXLUDE_VARS
```

Comment:
```suggestion
    _EXCLUDE_VARS: t.ClassVar[tuple[str, ...]] = ("self", "extra_fields", "extra")
    extra: dict[str, t.Any]

    @classmethod
    def _supported_fields(cls) -> list[str]:
        signature = inspect.signature(cls.__init__)
        return [
            name for name in signature.parameters.keys() if name not in cls._EXCLUDE_VARS
```

Reply:
At first, I read this and thought "oh no, how did this pass tests"?

And then I saw that I made the same typo twice! Thanks for spotting this.